src/components/ngRedux.js

```diff
@@ -42,45 +48,56 @@ export default function ngReduxProvider() {
   };
 
   this.$get = ($injector) => {
-    const resolveMiddleware = middleware => isString(middleware)
-      ? $injector.get(middleware)
-      : middleware;
+    if (_providedStore) {
+      const emptyStore = createNgReduxStore($injector, [], [], state => state, undefined);
```

Comment:
This pass through reducer is throwing errors for me. It's expecting it to be a well formed action object `{type: 'foobar'}`.

Changed to `(state, action) => action.payload` and then changed line 6 of the store wrapper to just dispatch as:

```
ngReduxStore.dispatch({type:'@@NGREDUX_PASSTHROUGH', payload: newState});
```

Reply:
Ah! nice! I figured that might cause some issues.